### ISSUE TITLE
Figure out data hierarchy for displaying in UI

### ISSUE THREAD

**Issue by sethetter** [openwichita/public-meetings#40]
For example, right now we have "Meeting Types" that each have a `type` property. The value of this property is something like `council` or `boe` (board of education). But the MeetingType `title` prop is the specific meeting withing the more broad `type` property (there are several `boe` meeting types).

The question is -- What's the top level that we display to users? It seems like the `type` column on the MeetingType is what we need, but we don't have a friendly display for that right now (they're all just lowercase, and some are acronyms). Having descriptions for these as well would be good.

The only thing I can really come up with is creating a `Category` model to hold this, and give it a `slug` property to associate with the `type` column on `MeetingTypes`.

@Mearnest @aaronarduino thoughts?

**Comment by aaronarduino** [comment]
Seems reasonable to me. I think refining things to be more descriptive and understandable would be good as well.

**Comment by infernocloud** [comment]
Yeah we definitely need a "category" that sits above "organizations" or "meeting types". I included a few examples of these in my design doc: https://paper.dropbox.com/doc/Open-Wichita-Public-Meetings-phUWrX8PJuUpfroQ56krE

**Comment by sethetter** [comment]
@infernocloud the doc is looking good! Lots of great ideas in there.

It seems like it would make the most sense for the `MeetingType` model to be named `Meeting` and `MeetingType` be used for these proposed categories.

> There are only two hard things in Computer Science: cache invalidation and naming things. -- Phil Karlton

**Comment by infernocloud** [comment]
In my mind, it makes sense to even split it up to one more level.
- Organization: Pachyderm Club
- Category: Republican Party/Political
- Meeting: [Pachyderm Luncheon, Special Voting Session, etc.]

Category lets us group and find similar organizations. Organization lets us easily see all scheduled events for the same entity, and then the meetings would be unique meeting "types" that then have a schedule associated with them. I'm thinking its possible for an organization to have more than 1 meeting "type". This _may_ be a lot of changes right now though. But it is something to keep in mind.

**Comment by sethetter** [comment]
@infernocloud I agree, organizations should also be represented here. Might be best to make it a many-to-many relationship just in case some meetings involve multiple orgs.

What you mentioned is a decent amount of change but also better to handle data structure issues early on.

@Mearnest what's your opinion on all this?

**Comment by Mearnest** [comment]
@sethetter Michael's suggestion for organization makes sense. I can add this to the Launch list.
